vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Install and use packages on Linux with vcpkg
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 08/23/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages on a Linux environment using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages on Linux with vcpkg
+
+This tutorial shows you how to set up a C++ "Hello World" CMake project using the `fmt` library. Manage dependencies, construct the application, and execute the project with the help of vcpkg. This guide is tailored for a Linux environment.
+
+## Prerequisites
+
+- A terminal to work in.
+- `git`, `CMake`, and a C++ compiler installed on your system
+- vcpkg installed on your system and on your PATH
+
+## 1 - Set up the project
+
+1. Create the project directory
+
+    This tutorial assumes your project is located in `/home/helloworld`. If you intend to place your project somewhere else, replace any references to that path with your preferred path.
+
+    Create a directory named "helloworld" to contain your project files, put it right next to your vcpkg directory. Then, navigate to your project directory.
+
+    ```bash
+    mkdir helloworld && cd helloworld
+    ```
+
+2. Create the manifest
+
+    Next, create a file named `vcpkg.json` in your project's directory. This is your manifest file. vcpkg reads the manifest file to learn what dependencies to install.
+
+    From within the `helloworld` directory, run:
+
+    ```bash
+    vcpkg new --application
+    ```
+
+    The `vcpkg new` command adds a `vcpkg.json` file and a `vcpkg-configuration.json` file to your projects directory. The `vcpkg.json` file should look like this:
+
+    ```json
+    {}
+    ```
+
+    The `vcpkg.json` file serves as the foundation to integrate vcpkg in your C++ project. Initially, the file contains an empty JSON object; additional attributes like dependencies and supported features can be added as your project grows.
```

Comment:
What is a 'foundation to integrate vcpkg in your C++ project'  Foundation? I'd drop that mention and just say "The vcpkg.json file integrates vcpkg into your C++ project by providing a list of dependencies and supported features that are part of your project"